templates/pages/reset-password.jsx

```diff
@@ -1,10 +1,56 @@
 var React = require('react');
 
+var Header = require('../components/header/header.jsx');
+var IconText = require('../components/icontext.jsx');
+var ResetView = require('../components/reset-password-view.jsx');
+var RequestView = require('../components/request-reset-view.jsx');
+var Router = require('react-router');
+
+// This wraps every view
 var ResetPassword = React.createClass({
+  getInitialState: function() {
+    return {
+      submitForm: false,
+      email: true
+    };
+  },
   render: function() {
     return (
-      <div>Kill me please</div>
+      <div>
+        <Header redirectText="Need an account?" redirectLabel="Sign up" redirectPage="signup" />
+
+        <div>
+            {!this.state.submitForm && !this.state.email ?
+              <RequestView submitForm={this.handleResetPassword}/> : '' }
```

Comment:
`: false` means it won't even inject *anything*, rather than injecting an empty string which we're doing now